src/forwarder/miniforwarder.cpp

```diff
@@ -306,7 +306,12 @@ void miniforwarder::end_readout(const YAML::Node& n) {
         return;
     }
 
-    _notification->block(_mode, image_id, _folder);
+    try {
+        _notification->block(_mode, image_id, _folder);
+    } catch (L1::CannotFetchPixel& e) {
+        LOG_CRT << "Block failed because exception occurred.";
```

Comment:
Would it be possible to make the log message more specific?

Of course, I don't know system's ins and outs, but the message seems to be a bit too generic considering that it will be emitted only when (judging from its name) a rather specific exception happened.

Reply:
This is a secondary message that gets printed after a primary message about the specifics in the _notification->block code.  The primary message is very specific.